app/hooks/use-display-currency.ts

```diff
@@ -53,6 +54,10 @@ export const useDisplayCurrency = () => {
     )
   }, [dataCurrencyList, displayCurrency])
 
+  const fractionDigits = useMemo(() => {
```

Comment:
It takes longer to memoize this then it does to just have `const fractionDigits = displayCurrencyInfo.fractionDigits` which is a simple assignment vs a function call to useMemo every render where it has to check for equality of dependencies.